pvlib/solarposition.py

```diff
@@ -1210,3 +1210,50 @@ def hour_angle(times, longitude, equation_of_time):
     )).total_seconds() / 3600. for t in times])
     timezone = times.tz.utcoffset(times).total_seconds() / 3600.
     return 15. * (hours - 12. - timezone) + longitude + equation_of_time / 4.
+
+
+def _hours(times, hour_angle, longitude, equation_of_time):
```

Comment:
I'd like a more descriptive function name or a one line documentation for `_hours` and `_times`.